streams/src/test/java/org/apache/kafka/streams/state/internals/CachingPersistentWindowStoreTest.java

```diff
@@ -119,29 +126,24 @@ public void closeStore() {
         cachingStore.close();
     }
 
-    @SuppressWarnings("deprecation")
+    @SuppressWarnings({"deprecation", "unchecked"})
     @Test
     public void shouldDelegateDeprecatedInit() {
-        final WindowStore<Bytes, byte[]> inner = EasyMock.mock(WindowStore.class);
+        final WindowStore<Bytes, byte[]> inner = mock(WindowStore.class);
         final CachingWindowStore outer = new CachingWindowStore(inner, WINDOW_SIZE, SEGMENT_INTERVAL);
-        EasyMock.expect(inner.name()).andStubReturn("store");
-        inner.init((ProcessorContext) context, outer);
-        EasyMock.expectLastCall();
-        EasyMock.replay(inner);
+        when(inner.name()).thenReturn("store");
         outer.init((ProcessorContext) context, outer);
-        EasyMock.verify(inner);
+        verify(inner).init(eq((ProcessorContext) context), eq(outer));
```

Comment:
Can you please double check if `eq()` matchers are needed here?

Reply:
Matchers are indeed not needed. Fixed in upcoming commits.

Reply:
Also experienced same with https://github.com/apache/kafka/pull/12524, `eq` was either not necessary or even caused tests to break